modules/appeals_api/app/models/concerns/appeals_api/appeal_model.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module AppealModel
+    extend ActiveSupport::Concern
+
+    included do
+      scope :without_status_updates_since, lambda { |time|
+        status_update_table = AppealsApi::StatusUpdate.table_name
+        join_clause = <<~JOIN
+          LEFT JOIN #{status_update_table}
+          ON #{table_name}.id = CAST(#{status_update_table}.statusable_id as uuid)
+          AND #{status_update_table}.id IS NULL
+          AND #{status_update_table}.statusable_type = '#{sanitize_sql(klass.name)}'
+        JOIN
+        where("#{table_name}.updated_at <= ?", time)
+          .where.not(id: joins(join_clause).where("#{status_update_table}.created_at >= ?", time).distinct.pluck(:id))
+      }
+
+      scope :with_pii, -> { where.not(form_data_ciphertext: nil).or(where.not(auth_headers_ciphertext: nil)) }
+
+      scope :with_expired_pii, lambda {
```

Comment:
Cases (1), (2), and (3) below are intended to match up to the acceptance criteria:

> Appeal is in any status and its status was last changed more than 45 days ago (check `AppealsApi::StatusUpdate` table)
>
> Appeal is in the "success" or "complete" status and its status was last changed more than 7 days ago (check `AppealsApi::StatusUpdate` table)
>
> Appeal is in the "error" status, its error detail contains the words "Unidentified Mail," and its status was last changed more than 7 days ago (check `AppealsApi::StatusUpdate` table)